libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -608,34 +653,41 @@ export class PackageExtractor {
     options: IExtractorOptions,
     state: IExtractorState
   ): Promise<void> {
-    const { includeNpmIgnoreFiles, targetRootFolder } = options;
-    const { projectConfigurationsByPath, archiver } = state;
+    const { includeNpmIgnoreFiles, targetRootFolder, terminal } = options;
+    const {
+      projectConfigurationsByPath,
+      packageJsonByPath,
+      dependencyConfigurationsByNameAndVersion,
+      archiver
+    } = state;
     let useNpmIgnoreFilter: boolean = false;
-    let includeFilters: IMinimatch[] | undefined;
-    let excludeFilters: IMinimatch[] | undefined;
 
     const sourceFolderRealPath: string = await FileSystem.getRealPathAsync(sourceFolderPath);
     const sourceProjectConfiguration: IExtractorProjectConfiguration | undefined =
       projectConfigurationsByPath.get(sourceFolderRealPath);
-    if (sourceProjectConfiguration) {
-      if (!includeNpmIgnoreFiles) {
-        // Only use the npmignore filter if the project configuration explicitly asks for it
-        useNpmIgnoreFilter = true;
-      }
-      if (sourceProjectConfiguration.patternsToInclude?.length) {
-        includeFilters = sourceProjectConfiguration.patternsToInclude.map(
-          (p) => new Minimatch(p, { dot: true })
-        );
+
+    const packagesJson: IPackageJson | undefined = packageJsonByPath.get(sourceFolderPath);
```

Comment:
Use the `sourceFolderRealPath` instead. Looks like that is what's used when it's originally added to the map